commands/bench/lib/bench-all-cumulus.sh

```diff
@@ -74,20 +76,57 @@ run_cumulus_bench() {
 echo "[+] Compiling benchmarks..."
 cargo build --profile $profile --locked --features=runtime-benchmarks -p polkadot-parachain-bin
 
-# Assets
-run_cumulus_bench assets asset-hub-westend
-run_cumulus_bench assets asset-hub-rococo
+# Run benchmarks for all pallets of a given runtime if runtime argument provided
+get_arg optional --runtime "$@"
+runtime="${out:-""}"
+
+if [[ $runtime ]]; then
+  paraId=""
+  case "$runtime" in
+    asset-*)
+      category="assets"
+    ;;
+    collectives-*)
+      category="collectives"
+    ;;
+    coretime-*)
+      category="coretime"
+    ;;
+    bridge-*)
+      category="bridge-hubs"
+    ;;
+    contracts-*)
+      category="contracts"
+    ;;
+    glutton-*)
+      category="glutton"
+      paraId="1300"
+    ;;
+    *)
+      echo "Unknown runtime: $runtime"
+      exit 1
+    ;;
+  esac
+
+  run_cumulus_bench $category $runtime $paraId
+
+  exit 0
```

Comment:
@seadanda this is the reason it doesn't go further and commit weights. just remove this exit 0

Reply:
ahh is this sourced into something else rather than run as a standalone script?

Reply:
yea, it's part of bench.sh, so supposed that it goes further to collect a diff
https://github.com/paritytech/command-bot-scripts/blob/main/commands/bench/bench.sh#L99
as this is common for all kinds of benches

Reply:
btw i removed and re-started bench to test it